### ISSUE TITLE
Maximum Flashloan amount on dodo

### ISSUE THREAD

**Issue by dani-s-92** [yuichiroaoki/poly-flash#58]
Hello,

you do not check the maximum flashloan amount you can get from dodo if I read your code correctly. 
If I am wrong can u tell me where you do that? 

If not it would be necessary to do it I think, because now your trade can get reverted because the flashloan you are asking for is bigger than the max available flashloan isn't it?

Best regards

**Comment by yuichiroaoki** [comment]
Yes, correct. You should check the token balance of the dodo pool you're going to borrow from. 
However, maybe using balancer flashloan is better than dodo in terms of every aspect like pool size, kinds of tokens available, contract interface, etc.
